esteid/signing/views.py

```diff
@@ -137,21 +159,32 @@ def finish_session(self, request: "RequestType", *args, **kwargs):
         Checks the status of a signing session and attempts to finalize signing
         """
         signer_class = self.select_signer_class()
-        signer = signer_class.load_session(request.session)
+        self._signer_instance = signer_class.load_session(request.session)
```

Comment:
hmm, wouldn't this break with async workers? e.g. wouldn't self be shared between multiple requests?

(i am a bit rusty on django so just checking)

Reply:
Valid point. As we agreed not to merge it yet, I won't jump into doing any changes; once we understand the proper way forward keeping a stateful signer may no longer be required at all.